app/handler/updater.py

```diff
@@ -18,7 +18,7 @@ def run_sli_update():
     while True:
         time.sleep(int(os.getenv('UPDATE_INTERVAL_SECONDS', 600)))
         try:
-            for product in get_products():
+            for product in get_products(None):
```

Comment:
I believe I broke this when I added filter to the `get_products` endpoint. I think it would be easier to supply default parameters to the function itself than passing `None` as a parameter.